src/languages/scss.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  argumentMatcher,
+  cascadingMatcher,
+  conditionMatcher,
+  createPatternMatchers,
+  matcher,
+  patternMatcher,
+} from "../util/nodeMatchers";
+import {
+  NodeMatcherAlternative,
+  ScopeType,
+  SelectionWithEditor,
+} from "../typings/Types";
+import { patternFinder } from "../util/nodeFinders";
+import { getNodeRange, selectChildrenWithExceptions } from "../util/nodeSelectors";
+import { SyntaxNode } from "web-tree-sitter";
+
+const STATEMENT_TYPES = [
+  "apply_statement",
+  "at_rule",
+  "charset_statement",
+  "debug_statement",
+  "declaration",
+  "each_statement",
+  "error_statement",
+  "for_statement",
+  "forward_statement",
+  "function_statement",
+  "if_statement",
+  "import_statement",
+  "include_statement",
+  "keyframes_statement",
+  "media_statement",
+  "mixin_statement",
+  "namespace_statement",
+  "placeholder",
+  "rule_set",
+  "supports_statement",
+  "use_statement",
+  "warn_statement",
+  "while_statement",
+];
+
+const nodeMatchers: Partial<Record<ScopeType, NodeMatcherAlternative>> = {
+  // map, list not supported in tree-sitter version
+  ifStatement: "if_statement",
+  condition: conditionMatcher("condition"),
+  // Currently data-lang=^"tel" not treated as a statement
+  // a[data-lang^="tel"] {
+  //    color: rgba(0, 255, 0, 0.5);
+  // }
+
+  statement: STATEMENT_TYPES,
+  string: "string_value",
+  functionCall: "call_expression",
+  namedFunction: ["mixin_statement", "function_statement"],
+  // What does `functionName` do?
+  functionName: ["mixin_statement.name!", "function_statement.name!"],
+  comment: ["comment", "single_line_comment"],
+  argumentOrParameter: argumentMatcher("arguments", "parameters"),
```

Comment:
Currently not handling some of the basic shapes correctly. 

`clip-path: ellipse(115px 55px at 50% 40%);`
or 
`clip-path: polygon(50% 0%, 60% 40%, 100% 50%, 60% 60%, 50% 100%, 40% 60%, 0% 50%, 40% 40%);`

Reply:
Worth filing? Easy fix?

Reply:
Fixed